cmd/mattermost/commands/webhook.go

```diff
@@ -448,6 +457,68 @@ func showWebhookCmdF(command *cobra.Command, args []string) error {
 	return errors.New("Webhook with id " + webhookId + " not found")
 }
 
+func moveOutgoingWebhookCmd(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("TeamId is not specified")
+	}
+
+	if len(args) < 2 {
+		return errors.New("Outgoing Webhook information is not specified")
+	}
```

Comment:
The number of arguments can be checked directly on the command struct using `Args: cobra.ExactArgs(2),`